### DISCUSSION TITLE
Add shortcut to install composer

### DISCUSSION THREAD

**Discussion by alexander-schranz** [php/frankenphp#1035] [Ideas]
What do you think if run composer via `frankenphp composer` which downloads may the `composer.phar` into that directory to run it.

```bash
wget https://github.com/dunglas/frankenphp/releases/download/v1.2.5/frankenphp-mac-arm64
mv frankenphp-mac-arm64 frankenphp
chmod +x frankenphp

frankenphp composer install
```

**Comment by withinboredom** [comment]
Unrelated, but related, I'm wondering how long it will be until composer is built as a standalone binary now that https://github.com/crazywhalecc/static-php-cli exists. :)

**Reply by alexander-schranz** [reply]
I think it will run always through your used PHP version as it does checks on your Version and Extensions, which I think make totally sense.

**Comment by dunglas** [comment]
On one hand, this would be very convenient, on the other hand, it's out of scope for FrankenPHP (I want to keep it as minimal as possible). So I've no hard opinion (yet) here!
